applications/nrf_desktop/src/modules/ble_discovery.c

```diff
@@ -184,45 +218,78 @@ static void dis_read_completed(struct bt_gatt_dm *dm, void *context)
 
 	rp.single.handle = attr->handle + 1;
```

Comment:
Because `rp` is `static struct` and it should be initialized with constant values.
Note from `bt_gatt_read` documentation: 

> This procedure is asynchronous therefore the parameters need to remains valid while it is active.

Reply:
ok